prog/dftb+/lib_dftb/coulomb.F90

```diff
@@ -56,6 +57,10 @@ module coulomb
   !> Maximal argument value of erf, after which it is constant
   real(dp), parameter :: erfArgLimit = 10.0_dp
 
+  !> Chunk size to use when obtaining neighbors dynamically via an iterator
+  integer, parameter :: iterChunkSize = 1000
```

Comment:
Any reason for that particular value? Not a change request, but could this be tuned for performance?

Reply:
Is not really a conscious choice. Just something I thought would avoid repeated calls for the iterator for small and middle sized non-periodic systems and not to waste too much memory...
We could make it tunable by the user, for sure, though all related automatic allocations would have to be turned probably into explicit allocations.